portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -26,7 +28,32 @@
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("Hello Aymar!");
+    /*These lines used to return a Hello Aymar message*/
+    //response.setContentType("text/html;");
+    //response.getWriter().println("Hello Aymar!");
+    ArrayList<String> msg = new ArrayList<String>();
```

Comment:
Nit: Prefer to have references to interfaces (`List<String>` in this case). It makes the code more ready for change

Reply:
Changed message container to CommentData class